src/julia_gc.c

```diff
@@ -817,6 +817,12 @@ void InitBags(UInt initial_size, Bag * stack_bottom, UInt stack_align)
     max_pool_obj_size = jl_gc_max_internal_obj_size();
     jl_gc_enable_conservative_gc_support();
     jl_init();
+
+    // Import GAPTypes module to have access
+    // to GapObj abstract type.
+    // Needs to be done before setting any GC states
+    jl_eval_string("import GAPTypes");
```

Comment:
Shouldn't we perform at least some minimal error checking, and abort if this fails, with a helpful error message?

Conversely: Have you tried what happens if this fails (e.g. because the user removed `GAPTypes` manually)

Reply:
No, I have not checked it. But error checking would be a good idea.

Reply:
I am pretty sure this will throw an exception (see also `JULIAINTERFACE_EXCEPTION_HANDLER`). We probably should catch that exception and then call `Panic` if it occurs. And of course we should have tested that at least one (by forcefully removing GAPTypes from `~/.julia` before starting GAP; and once again when loading `GAP.jl`).